qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/RunTest.java

```diff
@@ -17,10 +17,10 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(
 // defining ordered by name features folder
-features = {"src/test/resources/features"},
+features = { "src/test/resources/features" },
 // defining the definition steps package
 glue = "org.openmrs.contrib.qaframework.automation", plugin = {
-		"html:target/index.html", "message:target/cucumber.ndjson"}, monochrome = true)
+				"html:target/index.html", "message:target/cucumber.ndjson" }, monochrome = true)
```

Comment:
Is this change part of the ticket requirement?

Reply:
IDE playing tricks on me, thanks imma fix that!